python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1279,14 +1397,30 @@ def renumber_edges_by_type(self):
     @classmethod
     def is_multigraph(cls, df):
         """
-        Return True if df has >1 of the same src, dst pair
+        Parameters
+        ----------
+        df : dataframe containing edge data
+
+        Returns
+        -------
+        True if df has >= 1 of the same src, dst pair
         """
         return cls._has_duplicates(df, [cls.src_col_name, cls.dst_col_name])
 
     @classmethod
     def has_duplicate_edges(cls, df, columns=None):
         """
-        Return True if df has rows with the same src, dst, type, and columns
+        test for duplicate edges in the dataframe
```

Comment:
I think I prefer the old description better.  "test for duplicate edges..." makes it sound more like a debugging assertion API or something IMO. I dunno, maybe not?  Either way, the old one seemed more descriptive.
```suggestion
        Return True if df has rows with the same src, dst, type, and columns
```

Reply:
done